docs/en/scanners.md

```diff
@@ -30,7 +30,8 @@ class MyKeyboard(KMKKeyboard):
             row_pins=self.row_pins,
             # optional arguments with defaults:
             columns_to_anodes=DiodeOrientation.COL2ROW,
-            interval=0.02,  # Debounce time in floating point seconds
+            interval=0.01, # How often the matrix is sampled
```

Comment:
```suggestion
            interval=0.01, # Matrix sampling interval in ms
```
Same for the other parameter lists. ("how often" means frequency btw, which is the inverse).